client/src/app/[locale]/(general)/contacts/form/config.ts

```diff
@@ -16,15 +16,15 @@ export const InitialValues = {
 
 export const emailData = (values: IEmailData) => ({
   subject: `Зворотній зв'язок для ${values.name} ${values.surname}`,
-  to: 'Foxalina31@gmail.com',
+  to: 'vladkostirya@gmail.com',
```

Comment:
I'm not sure that's a good approach. I suggest we reconsider. Why 49%? Wouldn't it be better to make it 50% - some kind of gap?